.pre-commit-config.yaml

```diff
@@ -37,7 +37,7 @@ repos:
       - id: clang-format
         name: clang-format
         description: Format files with ClangFormat.
-        entry: clang-format-10
+        entry: clang-format
```

Comment:
I'm not up to date with MoveIt2 development but aren't other repos switching to clang-format-14?

Reply:
I had to make some kind of change here b/c clang-format-10 is not available on Ubuntu 22, so the first CI run failed. Luckily clang-format-14 is the default version for Ubuntu 22 and it also works for the other ROS distros.

Reply:
You should explicitly state `clang-format-14` (as in other repos) to ensure that a fixed version is used - even if the default changes.